### ISSUE TITLE
Building of samples tool failed with latest Pravega version

### ISSUE THREAD

**Issue by SomeshJoshi19** [pravega/pravega-samples#215]
Building of samples tool fails with Pravega version `0.6.0-2391.97363b3` and higher. Until last tested Pravega version `0.6.0-2383.e6dd0fa`, the samples tool build was successful.

The error occurred during build process indicates: not being able to download the lombok package for using lombok logger instance and annotations.

```
../pravega-samples# ./gradlew clean installDist

 > Task :flink-connector-examples:compileJava 
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.

> Task :hadoop-connector-examples:compileJava 
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.

> Task :pravega-client-examples:compileJava FAILED
:pravega-client-examples:compileJava/root/work/github/pravega-samples/pravega-client-examples/src/main/java/io/pravega/example/streamcuts/StreamCutsCli.java:27: error: package lombok.extern.slf4j does not exist
import lombok.extern.slf4j.Slf4j;
                          ^
/root/work/github/pravega-samples/pravega-client-examples/src/main/java/io/pravega/example/streamcuts/StreamCutsCli.java:35: error: cannot find symbol
@Slf4j
 ^
  symbol: class Slf4j
/root/work/github/pravega-samples/pravega-client-examples/src/main/java/io/pravega/example/streamcuts/StreamCutsExample.java:44: error: package lombok does not exist
import lombok.Cleanup;
             ^
/root/work/github/pravega-samples/pravega-client-examples/src/main/java/io/pravega/example/streamcuts/StreamCutsExample.java:45: error: package lombok.extern.slf4j does not exist
import lombok.extern.slf4j.Slf4j;
                          ^
/root/work/github/pravega-samples/pravega-client-examples/src/main/java/io/pravega/example/streamcuts/StreamCutsExample.java:47: error: cannot find symbol
@Slf4j
 ^
  symbol: class Slf4j
/root/work/github/pravega-samples/pravega-client-examples/src/main/java/io/pravega/example/consolerw/ConsoleReader.java:35: error: package lombok does not exist
import lombok.Cleanup;

             ^
/root/work/github/pravega-samples/pravega-client-examples/src/main/java/io/pravega/example/consolerw/ConsoleReader.java:36: error: package lombok.extern.slf4j does not exist
import lombok.extern.slf4j.Slf4j;
                          ^
/root/work/github/pravega-samples/pravega-client-examples/src/main/java/io/pravega/example/consolerw/ConsoleReader.java:60: error: cannot find symbol
@Slf4j
 ^
  symbol: class Slf4j
/root/work/github/pravega-samples/pravega-client-examples/src/main/java/io/pravega/example/consolerw/ConsoleReader.java:323: error: cannot find symbol
.
.
.
.

  symbol:   variable log
  location: class BackgroundReader
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: /root/work/github/pravega-samples/pravega-client-examples/src/main/java/io/pravega/example/streamcuts/StreamCutsExample.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
28 errors
 FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':pravega-client-examples:compileJava'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 5.0.
See https://docs.gradle.org/4.5/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 14s
23 actionable tasks: 19 executed, 4 up-to-date
```

**Comment by SomeshJoshi19** [comment]
[error.log](https://github.com/pravega/pravega-samples/files/3733293/error.log)

**Comment by RaulGracia** [comment]
Fixed via #217.
